test/e2e/suit_test.go

```diff
@@ -55,13 +55,18 @@ var _ = ginkgo.SynchronizedBeforeSuite(func() []byte {
 }, func(bytes []byte) {
 	kubeconfig = os.Getenv("KUBECONFIG")
 	gomega.Expect(kubeconfig).ShouldNot(gomega.BeEmpty())
+
 	config, err := framework.LoadRESTClientConfig(kubeconfig, hostContext)
 	gomega.Expect(err).ShouldNot(gomega.HaveOccurred())
+
 	restConfig = config
 	kubeClient, err = kubernetes.NewForConfig(restConfig)
 	gomega.Expect(err).ShouldNot(gomega.HaveOccurred())
-	clusterLinkClient, err = versioned.NewForConfig(restConfig)
-	gomega.Expect(err).ShouldNot(gomega.HaveOccurred())
```

Comment:
suggest keep it

Reply:
kosmosclient and clusterlinkclient are the same, it is better to use the former uniformly.